setup.py

```diff
@@ -1376,7 +1376,11 @@ def run(self):
             # What executable to use?  This one I guess.
             os.spawnl(os.P_NOWAIT, sys.executable,
                       sys.executable, filename,
-                      "-quiet", "-wait", str(os.getpid()), "-install")
+                      "-install",
+                      "-destination {}".format(self.install_lib),
```

Comment:
Is it possible `self.install_lib` has a space in it?

Reply:
Good question. Will need to give it a try.

Reply:
Good catch @mhammond !

```
D:\User\GIT\pywin32>"C:\Program Files\Python37\python.exe" .\pywin32_postinstall.py -install -destination C:\Program Files (x86)
usage: pywin32_postinstall.py [-h] [-install] [-remove] [-wait WAIT] [-silent]
                              [-quiet] [-destination DESTINATION]
pywin32_postinstall.py: error: argument -destination: Path "C:\Program" does not exist!

D:\User\GIT\pywin32>"C:\Program Files\Python37\python.exe" .\pywin32_postinstall.py -install -destination "C:\Program Files (x86)"
Parsed arguments are: Namespace(destination='C:\\Program Files (x86)', install=True, quiet=False, remove=False, silent=False, wait=None)
```